docs/errors/could-not-authorize-user.md

```diff
@@ -0,0 +1,42 @@
+---
+id: could-not-authorize-user
+title: Could not authorize user
+sidebar_label: Could not authorize user
+slug: /errors/could-not-authorize-user
+---
+
+# Could not authorize user
+
+## Message
+
+»   Error: Could not authorize user
+
+## What went wrong?
+
+Amplication CLI works with the hosted version on https://app.amplication.com. It's possible that you might have run the command 
+
+```$ amp config:set AMP_SERVER_URL http://localhost:3000```
+
+while going through the documentation.
+
+
+## What to do?
+
+Please run the command `amp config`. You should get below output stating AMP_SERVER_URL=http://localhost:3000/
+
+AMP_CURRENT_APP=undefined
+AMP_CURRENT_ENTITY=undefined
+AMP_CURRENT_FIELD=undefined
+AMP_SERVER_URL=http://localhost:3000/
+AMP_OUTPUT_FORMAT=undefined
+
+This means that AMP_SERVER_URL is set to http://localhost:3000/ We need to set the URL to https://app.amplication.com.
+
+Run the below command to resolve this issue.
+
+```$ amp config:set AMP_SERVER_URL https://app.amplication.com```
+
+After running the above command the issue should be resolved. Post which you may run the command `amp auth TOKEN` with the token generated from the Amplication server UI.
```

Comment:
Run the command `amp config` to see the URL of the Amplication server used by the CLI.

The results should look like the below
```
AMP_SERVER_URL=http://localhost:3000/
AMP_CURRENT_APP=undefined AMP_CURRENT_ENTITY=undefined AMP_CURRENT_FIELD=undefined AMP_SERVER_URL=http://localhost:3000/ AMP_OUTPUT_FORMAT=undefined
```

Use this command to set the correct URL for the Amplication server 
```
$ amp config:set AMP_SERVER_URL https://app.amplication.com
```